packages/contracts/.gas-snapshot

```diff
@@ -1,12 +1,12 @@
-DepositFeedTest:test_depositTransaction_ContractCreationReverts() (gas: 11352)
-DepositFeedTest:test_depositTransaction_NoValueContract() (gas: 21076)
+DepositFeedTest:test_depositTransaction_ContractCreationReverts() (gas: 11233)
+DepositFeedTest:test_depositTransaction_NoValueContract() (gas: 20962)
 DepositFeedTest:test_depositTransaction_NoValueEOA() (gas: 21427)
-DepositFeedTest:test_depositTransaction_createWithZeroValueForContract() (gas: 21094)
+DepositFeedTest:test_depositTransaction_createWithZeroValueForContract() (gas: 20980)
 DepositFeedTest:test_depositTransaction_createWithZeroValueForEOA() (gas: 21467)
-DepositFeedTest:test_depositTransaction_withEthValueAndContractContractCreation() (gas: 28117)
-DepositFeedTest:test_depositTransaction_withEthValueAndEOAContractCreation() (gas: 25856)
-DepositFeedTest:test_depositTransaction_withEthValueFromContract() (gas: 27785)
-DepositFeedTest:test_depositTransaction_withEthValueFromEOA() (gas: 25817)
+DepositFeedTest:test_depositTransaction_withEthValueAndContractContractCreation() (gas: 28003)
+DepositFeedTest:test_depositTransaction_withEthValueAndEOAContractCreation() (gas: 25031)
+DepositFeedTest:test_depositTransaction_withEthValueFromContract() (gas: 27671)
+DepositFeedTest:test_depositTransaction_withEthValueFromEOA() (gas: 28407)
```

Comment:
I'm not sure why this test in particular got so much more expensive. Will dig into it, but for now I think we should just make sure we're checkpointing this so that we can begin to improve on it.

FWIW, the previous snapshot was pretty old, so the increases are necessarily associated with this PR.

Reply:
The call itself is still below 4500, so I think it's changes in the test file most likely:
```
Running 1 test for contracts/test/DepositFeed.t.sol:DepositFeedTest
[PASS] test_depositTransaction_withEthValueFromEOA() (gas: 28407)
Traces:
  [28407] DepositFeedTest::test_depositTransaction_withEthValueFromEOA()
    ├─ [0] VM::prank(DepositFeedTest: [0xb4c79dab8f259c7aee6e5b2aa729821864227e84], DepositFeedTest: [0xb4c79dab8f259c7aee6e5b2aa729821864227e84])
    │   └─ ← ()
    ├─ [0] VM::expectEmit(true, true, false, true)
    │   └─ ← ()
    ├─ emit TransactionDeposited(from: DepositFeedTest: [0xb4c79dab8f259c7aee6e5b2aa729821864227e84], to: 0x0000000000000000000000000000000000000001, mint: 100, value: 0, gasLimit: 50000, isCreation: false, data: 0x1111)
    ├─ [4449] Target::depositTransaction{value: 100}(0x0000000000000000000000000000000000000001, 0, 50000, false, 0x1111)
    │   ├─ emit TransactionDeposited(from: DepositFeedTest: [0xb4c79dab8f259c7aee6e5b2aa729821864227e84], to: 0x0000000000000000000000000000000000000001, mint: 100, value: 0, gasLimit: 50000, isCreation: false, data: 0x1111)
    │   └─ ← ()
    └─ ← ()
```